deploy/central/tasks-chart/templates/task-ods-build-typescript.yaml

```diff
@@ -66,6 +66,16 @@ spec:
       description: "Node.js version to use - supported versions: 16"
       type: string
       default: "16"
+    - name: build-dir
+      description: >-
+        The directory into which `npm run build` places files.
+        The files inside `build-dir` will be copied to `output-dir`.
+      type: string
+      default: "dist"
```

Comment:
What are the defaults for common FE frameworks? Is `dist` the best default?

Reply:
I think `dist` will be the best option as by default Angular and vanilla TypeScript will build to `dist/` while React will build to `build/` and Svelte to `public/`.

Reply:
Cool! Maybe we can mention this in the doc? Like: "other common build directories are `build/` and `public`"